src/Features/VisualBasic/Completion/KeywordRecommenders/Statements/ElseKeywordRecommender.vb

```diff
@@ -45,5 +44,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.KeywordRecommenders.Stat
 
             Return SpecializedCollections.EmptyEnumerable(Of RecommendedKeyword)()
         End Function
+
+        Private Function IsDirectlyInIfOrElseIf(context As VisualBasicSyntaxContext) As Boolean
+            ' Maybe we're after the Then keyword
+            If context.TargetToken.IsKind(SyntaxKind.ThenKeyword) AndAlso
+                context.TargetToken.Parent?.Parent.IsKind(SyntaxKind.MultiLineIfBlock, SyntaxKind.ElseIfBlock) Then
+                Return True
```

Comment:
If the TargetToken is the ThenKeyword, then can bad things happen if you fall through to the code below? Doesn't look like the code below is set up to deal with the ThenKeyword.